paddle/operators/elementwise_mod_op.cu

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+#define EIGEN_USE_GPU
+#include "paddle/operators/elementwise_mod_op.h"
+
+namespace ops = paddle::operators;
+
+REGISTER_OP_GPU_KERNEL(
+    elementwise_mod,
+    ops::ElementwiseModKernel<paddle::platform::GPUPlace, int>);
```

Comment:
Let it be `int64`

Reply:
Since our python test is int32, here should be int. And many of our operators need to support multi-types. Duplicate the different types need to be supported in a more elegant manner.
https://github.com/PaddlePaddle/Paddle/issues/5646